programs/conditional_vault/src/lib.rs

```diff
@@ -106,41 +109,48 @@ pub mod conditional_vault {
         let on_finalize_token_symbol = format!("p{}", underlying_token_symbol);
```

Comment:
@jshiohaha Random question about token metadata. If the token symbol is just `p{}` so `pMeta` won't that make it hard to tell apart / search for the different proposal tokens if there are 10s of `pMeta`s, even if the toplevel metadata name is different? Aside from aesthetics is there a downside to having the ticker as pMeta_{proposal_number}?

Reply:
Sure, the underscore does mess with the aesthetics. Probably best to leave as is for now and just see how annoying / not annoying it looks once there's been a few proposals up using the new contracts.